etc/kayobe/ansible/cis.yml

```diff
@@ -1,8 +1,10 @@
 ---
 
 - name: Security hardening
-  hosts: overcloud
+  hosts: cis-hardening
```

Comment:
Should the group_vars file be moved from overcloud to cis-hardening?

Reply:
Good call - I've now moved them